tests/conftest.py

```diff
@@ -18,6 +18,12 @@ def cwd():
     os.chdir(root)
 
 
+@pytest.fixture(scope="class")
+def experiment_dir(cwd):
+    os.chdir('tests/experiment')
+    yield
```

Comment:
Good catch. I was thinking the `cwd` fixture takes care of that, but it does so before tests that use it, not after.